9.24 RN

Adam-Dupaski opened the pull request.

Commit:
Add RN

Commit:
Add RN

Commit:
Edit RN

Yiyun333 committed:
Language review and add a cross-reference

Commit:
Update for KI

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

atennapel commented on content/en/docs/releasenotes/studio-pro/9/9.24.md:
```diff
@@ -0,0 +1,146 @@
+
+* We deprecated `Core.isSubClassOf` method variants. You can instead use the `IMendixObject.isInstanceOf(IMetaObject)`, `IMendixObject.isInstanceOf(String)`, `IMetaObject.isSubClassOf(IMetaObject)`, or `IMetaObject.isSubClassOf(String)` methods to achieve the same result.
+* We deprecated the setter methods that were for internal use. We have provided constructors as an alternative for these methods whereever applicable.
+* We deprecated the `com.mendix.core.Core.deleteAsync` methods. You can use the synchronous version or run the synchronous version in a thread if needed. In addition, we introduced a delete method with events and with the `useDeleteBehavior` parameter as a replacement for calling `deleteAsync` with `useDeleteBehavior` set to false.
```
We (Runtime Core team) would like to adjust this change by changing "run the synchronous version in a thread if needed" to "run the synchronous version in a task queue if needed"

Adam-Dupaski replied:
Updated, thanks: https://github.com/mendix/docs/pull/5759/commits/4094f7878ac0b1d035863b1bd31dd87c4f0cd007

atennapel replied:
Thanks!

Commit:
Update via comment

Robinfr commented on content/en/docs/releasenotes/studio-pro/9/9.24.md:
```diff
@@ -0,0 +1,146 @@
+
+#### Other Improvements
+
+* Studio Pro now now retrieves non-persistable objects over a direct association without sending a request as long as they have no virtual attributes.
```
The client does this, not Studio Pro.

Adam-Dupaski replied:
Changed, thanks: https://github.com/mendix/docs/pull/5759/commits/6bca2bc4aef9b92954d3ed1e77279ce2514a10aa

Commit:
Change via comment

MariaShaposhnikova committed:
Review the RN about App from spreadsheet

ConnorLand committed:
Add anchor link for Never option

Commit:
Add via Slack

Yiyun333 committed:
Link to the converting micro/nanoflow feature

Commit:
Add RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Edit RN

Commit:
Update

ConnorLand committed:
Add snippet link

Commit:
Proofread

MariaShaposhnikova committed:
Add a cross-reference for Gradle setting

Commit:
Add via Slack

Commit:
Update

Commit:
Add links to business events docs

Commit:
Merge pull request #5795 from mendix/jh-addbelinks

Add links to business events docs

Commit:
Add cross-reference

Commit:
Add cross-reference

Commit:
Merge branch 'development' into ad-9.24-rn

Commit:
Update for KIs

Commit:
Update for KIs

ConnorLand committed:
Add attribute feature change

This improvement does not require a cross reference, but is accompanied by some new doc additions in the PR linked below.

Please see this comment in this PR for more information: https://github.com/mendix/docs/pull/5781#issuecomment-1482648220

ConnorLand committed:
Incorporate Feedback

ConnorLand committed:
Remove duplicated bullet

The doc improvements are already documented in the entire " Selecting Attributes" section.
